tests/integration/docusaurus/setup/airflow_cloud_integration.py

```diff
@@ -0,0 +1,45 @@
+import os
+from datetime import datetime
+
+from airflow import DAG
+from airflow.operators.python import PythonOperator
+
+import great_expectations as gx
+
+# Replace <YOUR_ACCESS_TOKEN> and <YOUR_CLOUD_ORGANIZATION_ID> with your credentials.
+# To get your user access tokesn and organization ID, see:
+# (https://docs.greatexpectations.io/docs/cloud/set_up_gx_cloud#get-your-user-access-token-and-organization-id).
+
+GX_CLOUD_ACCESS_TOKEN = os.environ.get("GX_CLOUD_ACCESS_TOKEN")
+GX_CLOUD_ORGANIZATION_ID = os.environ.get("GX_CLOUD_ORGANIZATION_ID")
+
+CHECKPOINT_NAME = "haebichan_checkpoint"
+
+
+# <snippet name="tests/integration/docusaurus/setup/airflow_cloud_integration.py run_gx_airflow">
+def run_gx_airflow():
+    context = gx.get_context(cloud_mode=True)
+    checkpoint = context.get_checkpoint(name=CHECKPOINT_NAME)
+    checkpoint.run()
```

Comment:
I would use an ephemeral context for this.
I don't think we get much by testing a `cloud_clontext` specifically.

If you need to test a cloud context specifically (which I would argue you do not), you should test against the containerized mercury backend instead of relying on pre-existing state in the production CloudAPI.
https://github.com/great-expectations/great_expectations/blob/d24bd37f248b09e401e0997bdf20cacf77b628b8/tasks.py#L798

What's the purpose of this test?